### PR TITLE
Infinite builds loop fix

### PR DESCRIPTION
**Pull request by ashfame**
In this PR, we decided to not drop a generated file into `src` dir for WP so as to avoid the situation of generated file re-triggering a build.

`schema.json` is now only generated under `schema` dir and copied to the right places under the build dir. And is now also configured to be ignored by webpack's watching for changes. 

This resolves the infinite loop of building assets.

### CODE REVIEW

File: webpack.config.cjs [hunk 1]
```diff
@@ -206,16 +207,6 @@ class EmitSubjectsSchemaPlugin {
 						execSync( './schema/build.mjs', { stdio: 'inherit' } );
 						const schema = readFileSync( SCHEMA_SRC );
 
-						// Write to WordPress plugin directory
-						await fs.promises.mkdir(
```
Comment (psrpinto) [inline webpack.config.cjs:210]: We can remove the `const fs = require( 'fs' );` at the top of the file as `fs` is no longer used

**Review by psrpinto — APPROVED** [review]
Thanks!

File: webpack.config.cjs (hunk 1 above)
Comment (ashfame) [reply webpack.config.cjs:210]: Already did, just pushing :)

**Comment by psrpinto** [comment]
I was thinking: Now that we found the way to exclude paths from the watcher, we could also have webpack copy the schema.json into `src/plugin`, assuming you want that to happen.

**Comment by ashfame** [comment]
@psrpinto Good point! I will try that in a separate PR :)

**Comment by psrpinto** [comment]
We ended up pushing that change in #148

**Pull request merged**